src/main/java/com/staxrt/tutorial/controller/PostController.java

```diff
@@ -0,0 +1,73 @@
+package com.staxrt.tutorial.controller;
+/* https://www.baeldung.com/spring-data-jpa-multiple-databases */
+
+import com.staxrt.tutorial.exception.ResourceNotFoundException;
+import com.staxrt.tutorial.model.Post;
+import com.staxrt.tutorial.model.User;
+import com.staxrt.tutorial.repository.PostRepository;
+import com.staxrt.tutorial.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/api/v2")
+
+public class PostController {
+
+    @Autowired
+    private PostRepository postRepository;
+
+    @GetMapping("/posts")
+    public List<Post> getAllPosts() {
+        return postRepository.findAll();
+    }
+
+    @GetMapping("/posts/{id}")
+    public ResponseEntity<Post> getPostById(@PathVariable(value = "id") Long postId)
+            throws ResourceNotFoundException {
+        Post post =
+                postRepository
+                        .findById(postId)
+                        .orElseThrow(() -> new ResourceNotFoundException("Post not found on :: " + postId));
+        return ResponseEntity.ok().body(post);
+    }
+    @PostMapping("/posts")
+    public Post createPost(@Valid @RequestBody Post post) {
+        return postRepository.save(post);
+    }
+    @PutMapping("/posts/{id}")
+    public ResponseEntity<Post> updatePost(
+            @PathVariable(value = "id") Long postId, @Valid @RequestBody Post postDetails)
+            throws ResourceNotFoundException {
+
+        Post post =
+                postRepository
+                        .findById(postId)
+                        .orElseThrow(() -> new ResourceNotFoundException("Post not found on :: " + postId));
+
+        post.setData(postDetails.getData());
+        post.setPrivacy(postDetails.getPrivacy());
+        post.setUpdatedAt(new Date());
+        final Post updatedPost = postRepository.save(post);
+        return ResponseEntity.ok(updatedPost);
+    }
+    @DeleteMapping("/user/{id}")
```

Comment:
Co to tu robi?

Reply:
Błąd którego nie wyłapałem, sugerowałem się kodem user i zamieniałem to na post a tutaj przeoczyłem

Reply:
No to może przy okazji napisz teścik do usuwania?